.github/CHANGELOG.md

```diff
@@ -506,12 +539,14 @@ and requirements-ci.txt (unpinned). This latter would be used by the CI.
 
 This release contains contributions from (in alphabetical order):
 
-Vishnu Ajith, Akash Narayanan B, Thomas Bromley, Olivia Di Matteo, Sahaj Dhamija, Tanya Garg, Josh Izaac,
-Prateek Jain, Ankit Khandelwal, Christina Lee, Ian McLean, Johannes Jakob Meyer, Romain Moyard, Esteban Payares,
-Pratul Saini, Maria Schuld, Arshpreet Singh, Ingrid Strandberg, Slimane Thabet, Antal Száva, David Wierichs,
-Vincent Wong.
+Vishnu Ajith, Akash Narayanan B, Thomas Bromley, Olivia Di Matteo, Sahaj
+Dhamija, Tanya Garg, Anthony Hayes, Theodor Isacsson, Josh Izaac, Prateek Jain,
+Ankit Khandelwal, Nathan Killoran, Christina Lee, Ian McLean, Johannes Jakob
+Meyer, Romain Moyard, Lee James O'Riordan, Esteban Payares, Pratul Saini, Maria
+Schuld, Arshpreet Singh, Ingrid Strandberg, Antal Száva, Slimane Thabet, David
+Wierichs, Vincent Wong.
```

Comment:
A massive contributor list 💯

Reply:
I just noticed that Jay is missing from this list, I recall him doing code review for bugs?